src/app/organisms/settings/Settings.jsx

```diff
@@ -112,6 +113,29 @@ function AppearanceSection() {
           content={<Text variant="b3">Hide nick and avatar change messages from room timeline.</Text>}
         />
       </div>
+      <div className="settings-appearance__card">
```

Comment:
Looks like you accidentally merged #564 into this PR.

Reply:
aaaaaaaaa